data/Li_2018/metadata.yml

```diff
@@ -0,0 +1,499 @@
+    were oven dried at 70 °C for at least 48 hr to obtain dry weight. Leaf PV traits
+    were estimated as described by Lenz, Wright, and Westoby (2006). The inverse of
+    leaf water potential (1/Psi leaf) was plotted against leaf relative water content
+    (RWCleaf, %) to facilitate calculation. Leaf TLP (MPa) was taken as the point
+    where the line became non-linear.
+- var_in: Ks
+  unit_in: kg/m/MPa/s
+  trait_name: sapwood_specific_conductivity
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates
+  methods: Stem vulnerability curves ^p Stem vulnerability curves were constructed
+    for each species by slowly dehydrating whole plants in 25-L pots (Martorell, Diaz-Espejo,
+    Medrano, Ball, & Choat, 2014; Sperry, Donnelly, & Tyree, 1988). Stem samples were
+    prepared for hydraulic measurement following Wheeler, Huggett, Tofte, Rockwell,
+    and Holbrook (2013) and Torres-Ruiz et al. (2015). Excising stem segments under
+    high xylem tension may allow air bubbles to enter the stem xylem even if performed
+    under water, consequently leading to an overestimation of the level of embolism.
+    To avoid this "cutting" artefact, stem samples were rehydrated for at least 2
+    hr before sample preparation by placing plants in darkness with the cut surface
+    submerged in water. Xylem tension was considered sufficiently relaxed when Psi leaf
+    was higher than -1 MPa; thereafter, a 10- to 15-cm stem segment was targeted for
+    xylem hydraulic conductivity measurements. Side branches were removed, and the
+    cut surfaces sealed with parafilm, if necessary. The first cut was made to the
+    upstream end of the branch, at least 30 cm from the targeted segment, and then
+    left to rehydrate for 10 min. Subsequent cuts were made at approximately 2-min
+    intervals, alternating between upstream and downstream ends of the segment. Once
+    the desired stem length was achieved, the sample was labelled and connected to
+    a pressure head (0.002 MPa) generated gravitationally, and the initial flow rate
+    (Kinitial) was measured using a digital liquid flow meter (Liqui-Flow L10, Bronkhorst
+    High-Tech BV, Ruurlo, Gelderland, The Netherlands) connected to flow analysis
+    programs FlowDDE and FlowPlot (Version 4.69 and 3.34, respectively, Bronkhorst,
+    FlowWare, http://downloads.bronhorst.com). Thereafter, the stem segment was attached
+    to a water tank filled with 2-mmol KCL solution. The segment was flushed with
+    a pressure of 200 kPa for at least 30 min to remove any embolism and then connected
+    again to the flow meter to measure final flowrate (Kfinal). Kfinal was normalized
+    by the pressure gradient, sapwood area, and stem length to obtain initial and
+    final xylem-specific hydraulic conductivity (Ks, kg m-1 s-1 MPa-1).
+- var_in: Amax
+  unit_in: umol{CO2}/m2/s
+  trait_name: leaf_photosynthetic_rate_per_area_maximum
```

Comment:
this trait should be `leaf_photosynthetic_rate_per_area_saturated`. Maximum is for when both CO2 and light are saturating. I'll change it.

Reply:
Lots of people mis-use terms or interchangeably use terms, which is hard.